### DISCUSSION TITLE
How to support class fields?

### DISCUSSION THREAD

**Discussion by xiaojundebug** [mdx-js/mdx#2091] [Q&A] [answered]
<img width="1008" alt="image" src="https://user-images.githubusercontent.com/24448924/181876498-d3d3bf5b-cb1a-4ba3-b590-b8ed662fdb7a.png">

---

<img width="1018" alt="image" src="https://user-images.githubusercontent.com/24448924/181876481-6c0c31ec-5cea-41c5-999a-6e1796eb1fbf.png">

**Comment by wooorm** [comment, marked as answer]
It’s not supported yet because it’s not supported in JavaScript yet (specifically the parser being used, Acorn).
Either don’t define them. Or don’t define them in MDX files but outside of MDX files.

**Reply by xiaojundebug** [reply]
thank you
